src/main/java/domain/Lotto.java

```diff
@@ -10,7 +11,7 @@
 
 public class Lotto {
 
-    public static final int NUMBERS_SIZE = 6;
+    private static final int NUMBERS_SIZE = 6;
```

Comment:
> 예를 들어 LottoMachine을 생성할때 Lotto class의 createLottoMachine 메서드로 생성하는 것이 아닌 , new 키워드로 생성하는데 numbersMaxSize값을 6이 아닌 다른 값으로 생성할 수 있겠다 싶더라고요!
> ```suggestion
>     private static final int NUMBERS_SIZE = 6;
> ```
> 
> 결론은 private으로 닫아 응집도가 높아졌지만 잠재적인 버그가 생길 가능성이 높아졌다 생각되는데 조앤의 생각은 어떤지 궁금합니다!


이 부분은 public으로 선언할지라도 사용처에서 그걸 사용하지 않고 임의의 값을 주입한다면 무조건 발생할 수 있는 문제 같아요. 
그래서 도메인 생성자 안에서 필요한 검증을 해주는 것이 중요하다고 생각하고요ㅎㅎ
지금도 Lotto의 생성자 안에서 검증을 잘 해주고 있으시기 때문에 저는 private이던 public이던 이 문제는 발생하진 않을거라고 생각해요!

Reply:
그렇네요!! 답변 감사합니다:)